node-server/package-lock.json

```diff
@@ -5,7 +5,6 @@
   "requires": true,
   "packages": {
     "": {
-      "name": "node-server",
```

Comment:
name-field has been removed. It is still defined in the package.json file, mistake (npm version?) or intentionally?